api/handlers/container/create_test.go

```diff
@@ -648,6 +650,50 @@ var _ = Describe("Container Create API ", func() {
 			Expect(rr.Body).Should(MatchJSON(jsonResponse))
 		})
 
+		It("should set PidMode and Privileged option", func() {
```

Comment:
We should split the tests for individual options as there is no commonality between these options.  Alternatively, we can write a single test for all HostConfig options